src/internal/SwitchBase.js

```diff
@@ -155,6 +155,10 @@ export default function createSwitch(
     }
   }
 
+   /**
+   ** NB: If changed, please update Checkbox, Switch and Radio
+   ** so that the API documentation is updated.
+   **/
```

Comment:
This comment block is causing one of your lint errors:

>/tmp/material-ui/src/internal/SwitchBase.js
  158:3  error  Delete `·`                                    prettier/prettier
  158:4  error  Expected space or tab before '*/' in comment  spaced-comment

Since the next comment is identical, you should remove it.